result/capture.yaml

```diff
@@ -9,34 +9,35 @@ components:
   schemas:
     Result.CaptureRequest:
       description: >-
-        The capture result request to the traffic generator 
+        The capture result request to the traffic generator.
+        Stops the port capture.
       type: object
+      required: [port_name]
       properties:
-        port_names:
+        port_name:
           description: >-
-            The names of objects to return results for.
-            An empty list will return all capture row results.
-          type: array
-          items:
-            type: string
+            The name of a port a capture was taken on.
+          type: string
           x-constraint:
           - '/components/schemas/Port/properties/name'
-        columns:
-          description: >-
-            The names of columns to return results for.
-            An empty list will return all columns.
-            The name column will always be included as it is the unique key.
-            The following is a description of the columns:
-              - name
-                  The name of a configured port
-              - packets
-                  The captured packets
-          type: array
-          items:
-            type: string
-            enum:
-              - name
-              - packets
 
+    Result.CaptureResult:
+      description: >-
+        The capture result from the traffic generator.
+      type: object
+      properties:
+        port_name:
+          description: >-
+            The name of the port the capture was taken on.
+          type: string
+        format:
+          description: >-
+            The requested format of the capture file.
+          type: string
+        capture:
```

Comment:
@ankur-sheth Do we serialize this ourselves?  or is there a way to provide a link to the capture file that the user can retrieve separately?  I guess we should probably brainstorm a bit on what is the ux here?  problem with providing a link is that we don't know how long we need to keep it on the server side.